pkg/converter/v2/converter.go

```diff
@@ -9,12 +9,20 @@ import (
 	"strings"
 )
 
+// AsyncapiVersion is the version of asyncapi that the document will be converted to.
 const AsyncapiVersion = "2.0.0-rc1"
 
+// Decode reads an asyncapi document from an input and stores it in the value.
```

Comment:
```suggestion
// Decode reads an AsyncAPI document from input and stores it in the value.
```
I'm wondering if it is `a value` or `the value` - do you mean any value or a particular one?